packages/block-editor/src/components/block-breadcrumb/breadcrumb-button.js

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
This `BlockBreadcrumbButton` is created just for demonstrating how the `PrimitiveButton` component could be used